Sage/carbon
# fix(popover): prevent from crashing when rendered in sidebar - FE-5621

Prevent all Popover based components like Select, ActionPopover, SplitButton, MultiActionButton and other from crashing the app when rendered in Sidebar

### Proposed behaviour

Fixes #5753 

### Checklist




#### QA


### Additional context



### Testing instructions
Replicate the steps shown in the video in #5753 issue and see if the error still persisrs.
Ignore the "Warning: Failed prop type: The prop `reference.current` is marked as required in `Popover`, but its value is `undefined`." error as it is not possible to fix in this ticket, this has to be fixed globally and there is an existing ticket for that
https://codesandbox.io/s/peaceful-haze-4jdmin
